Formula/soroban-cli.rb

```diff
@@ -1,7 +1,7 @@
 class SorobanCli < Formula
   desc "CLI for interacting with and deploying to Soroban"
   homepage "https://soroban.stellar.org"
-  url "https://github.com/stellar/soroban-tools.git", tag: "v0.9.4"
+  url "https://github.com/stellar/soroban-tools.git", tag: "v20.0.0-rc1"
   head "https://github.com/stellar/soroban-tools.git", branch: "main"
```

Comment:
the tag doesn't match `main`'s head; is that a problem?

Reply:
It shouldn't, these are distinct. This is what drives what gets installed on `brew --HEAD`, and the url is the default for a regular-ole `brew install`